static/src/javascripts/projects/common/modules/commercial/contributions-google-docs.js

```diff
@@ -8,7 +8,7 @@ export const epicGoogleDocUrl: string =
 export const bannerGoogleDocUrl: string =
     'https://interactive.guim.co.uk/docsdata/1CIHCoe87hyPHosXx1pYeVUoohvmIqh9cC_kNlV-CMHQ.json';
 export const epicMultipleTestsGoogleDocUrl: string = config.get('page.isDev')
-    ? 'https://interactive.guim.co.uk/docsdata-test/1VQ6yn2thnkFzjxIKt-AfOB_gJnX8omLNodkRyX7_Qbg.json'
+    ? 'https://interactive.guim.co.uk/docsdata/1THvo7I36Npb9GbKFAk6veIku3Hz5tBhpHobxrl0SoUc.json'
```

Comment:
I created a separate doc for testing locally

Reply:
worth making sure @jessewilkins knows that if he wants to test in CODE he'll have to use the test spreadsheet (and publish) and then lift & shift into the prod spreadsheet